Jenkinsfile

```diff
@@ -30,12 +30,28 @@ podTemplate(
             resourceLimitCpu: '2000m',
             resourceRequestMemory: '1000Mi',
             resourceLimitMemory: '2000Mi',
+        ),
+        ontainerTemplate(
+            name: 'docker',
+            image: 'docker:stable',
+            ttyEnabled: true,
+            command: 'cat',
```

Comment:
Add resource request and limit?

Reply:
do you have any preference? I'm not sure as I don't have any data points here.

Reply:
Can we collect some datas from Prometheus? If you don't have time for this, I think we can move on. It does not block this PR.

Reply:
Maybe this comment is missed, so I will move on.

> Can we collect some datas from Prometheus? If you don't have time for this, I think we can move on. It does not block this PR.